scalalib/api/src/ZincWorkerApi.scala

```diff
@@ -55,18 +55,28 @@ object Util {
     scalaVersion.startsWith("0.") ||
     scalaVersion.startsWith("3.")
 
-
-  def grepJar(classPath: Agg[os.Path], name: String, version: String, sources: Boolean = false) = {
-    val suffix = if (sources) "-sources" else ""
-    val mavenStylePath = s"$name-$version$suffix.jar"
-    val ivyStylePath = {
-      val dir = if (sources) "srcs" else "jars"
-      s"$version/$dir/$name$suffix.jar"
+  // eg, grepJar(classPath, name = "scala-library", versionPrefix = "2.13.")
+  // return first path in `classPath` that match:
+  // **/scala-library-2.13.*.jar or
+  // **/2.13.*/jars/scala-library.jar
+  def grepJar(classPath: Agg[os.Path], name: String, versionPrefix: String, sources: Boolean = false) = {
+    val suffix = if (sources) "-sources.jar" else ".jar"
+    lazy val dir = if (sources) "srcs" else "jars"
```

Comment:
The `lazy` is probably overkill.

Reply:
:) ahh. Yes. Don't know why I code like that 😁